client/components/mma/accountoverview/ProductCardConfiguration.ts

```diff
@@ -0,0 +1,43 @@
+import { palette } from '@guardian/source-foundations';
+import type { ProductTypeKeys } from '../../../../shared/productTypes';
+
+interface ProductCardConfiguration {
+	headerColor: string;
+	showBenefitsSection?: boolean;
+}
+
+export const productCardConfiguration: {
+	[productType in ProductTypeKeys]: ProductCardConfiguration;
+} = {
+	contributions: {
+		headerColor: palette.brand[600],
+	},
+	supporterplus: {
+		headerColor: palette.brand[500],
+		showBenefitsSection: true,
+	},
+	digipack: {
+		headerColor: palette.brand[500],
+	},
+	digitalvoucher: {
+		headerColor: '#ff5943',
```

Comment:
Could there be a work done to put these hex codes into the palette library? Would be good to have the brand colours all in one place...not sure who that work would sit with though

Reply:
If these colours are unique to MMA then they shouldn't be part of the Source colour palette. We could maybe add constants for these, but I've left them alone for the moment as Helene is still working on the final colour palette. Also, these colours aren't directly related to this PR, I've just extracted the config from the `ProductCard` component to tidy things up slightly as there's a lot going on in there!

Reply:
That's a good point re the Source colour palette which I didn't know about!